test/integration/helpers/Npm.cs

```diff
@@ -0,0 +1,73 @@
+﻿using OpenQA.Selenium;
+using System;
+using System.Diagnostics;
+using System.Linq;
+
+namespace Appium.Net.Integration.Tests.helpers
+{
+    internal class Npm
+    {
+
+        public static string GetNpmPrefixPath()
+        {
+
+            string npmPath = "npm";
+
+            if (Platform.CurrentPlatform.IsPlatformType(PlatformType.Windows))
+            {
+                npmPath = GetNpmExecutablePath();
+            }
+
+            string npmPrefixPath = RunCommand(npmPath, "-g root");
+
+            return npmPrefixPath.Trim();
+        }
+
+        private static string RunCommand(string command, string arguments)
+        {
+            int timeoutMilliseconds = 30000;
+            try
+            {
+                Process process = new Process
+                {
+                    StartInfo = new ProcessStartInfo
+                    {
+                        FileName = command,
+                        Arguments = arguments,
+                        RedirectStandardOutput = true,
+                        UseShellExecute = false,
+                        CreateNoWindow = true,
+                    }
+                };
+
+                process.Start();
+                string output = process.StandardOutput.ReadToEnd();
+                process.WaitForExit(timeoutMilliseconds);
+
+                return output;
+            }
+            catch (Exception ex)
+            {
+                Console.WriteLine($"Error executing command: {ex.Message}");
+                return null;
+            }
+        }
+
+        private static string GetNpmExecutablePath()
+        {
+            string result = RunCommand("where", "npm");
+            string npmPath;
+
+            string[] lines = result?.Split(new[] { Environment.NewLine }, StringSplitOptions.RemoveEmptyEntries);
+
+            npmPath = lines?.FirstOrDefault(line => line.EndsWith("npm.cmd"));
+
+            if (string.IsNullOrWhiteSpace(npmPath))
+            {
+                throw new ApplicationException("NPM path not found.");
```

Comment:
I believe this error message could be improved

Reply:
throw new NpmNotFoundException($"NPM executable not found at path: {npmPath}. Please make sure the NPM executable is installed and check the configured path.");